<?xml version="1.0" encoding="utf-8"?>
<TestRun id="2c5d745e-727a-4af6-b8a4-cdd5b86783f2" name="anna@cloudtest-win10 2023-01-22 19:07:42" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T19:07:42.6132957+00:00" queuing="2023-01-22T19:07:42.6132962+00:00" start="2023-01-22T19:07:32.8405335+00:00" finish="2023-01-22T19:07:42.6523717+00:00" />
  <TestSettings name="default" id="745b9e08-1419-4f6b-9c05-62facda6ffd4">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_19_07_42" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="3370227c-e4d0-47d6-9f5d-e7a3424b60eb" testId="25cf1ab5-8c91-2cb3-44d4-300ca8f4cbf6" testName="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_UpdateRowInParallel" computerName="cloudtest-win10" duration="00:00:03.8756282" startTime="2023-01-22T19:07:42.2897533+00:00" endTime="2023-01-22T19:07:42.2898257+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3370227c-e4d0-47d6-9f5d-e7a3424b60eb">
      <Output>
        <ErrorInfo>
          <Message>Assert.Equal() Failure&#xD;
Expected: 20&#xD;
Actual:   21</Message>
          <StackTrace>   at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_UpdateRowInParallel(Boolean extendedProtocol) in C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs:line 390&#xD;
   at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_UpdateRowInParallel() in C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs:line 100&#xD;
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_UpdateRowInParallel" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="25cf1ab5-8c91-2cb3-44d4-300ca8f4cbf6">
      <Execution id="3370227c-e4d0-47d6-9f5d-e7a3424b60eb" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureMembershipTableTests" name="MembershipTable_Azure_UpdateRowInParallel" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="25cf1ab5-8c91-2cb3-44d4-300ca8f4cbf6" executionId="3370227c-e4d0-47d6-9f5d-e7a3424b60eb" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" passed="0" failed="1" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.30]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.51]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.53]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:06.46]       Assert.Equal() Failure&#xD;
[xUnit.net 00:00:06.46]       Expected: 20&#xD;
[xUnit.net 00:00:06.46]       Actual:   21&#xD;
[xUnit.net 00:00:06.46]       Stack Trace:&#xD;
[xUnit.net 00:00:06.46]         C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs(390,0): at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_UpdateRowInParallel(Boolean extendedProtocol)&#xD;
[xUnit.net 00:00:06.46]         C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs(100,0): at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_UpdateRowInParallel()&#xD;
[xUnit.net 00:00:06.46]         --- End of stack trace from previous location ---&#xD;
[xUnit.net 00:00:06.50]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Min" timestamp="2023-01-22T19:07:42.2831710+00:00">
        <Text>[xUnit.net 00:00:06.46]     Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_UpdateRowInParallel [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T19:07:42.4612066+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>